README.md

```diff
@@ -54,6 +60,14 @@ status are checked every 15 minutes, you can do it with this configuration:
 0/15 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bin/rails decidim_vocdoni:change_election_status > /dev/null
 ```
 
+## Dynamic censuses
```

Comment:
The UI and the docs should talk the same language https://martinfowler.com/bliki/UbiquitousLanguage.html 

```suggestion
## Internal census
```